src/types.jl

```diff
@@ -36,35 +36,35 @@ Overload these if you want COBREXA to work with your own module types.
 $(EXPORTS)
 """
 module Accessors
-using ..ModuleTools
-@dse
-using ..Types
-using ..Internal.Macros
+    using ..ModuleTools
+    @dse
+    using ..Types
+    using ..Internal.Macros
 
-using SparseArrays
+    using SparseArrays
 
-"""
-    module Internal
+    """
+        module Internal
 
-Internal helpers for types.
+    Internal helpers for types.
 
-# Exports
-$(EXPORTS)
-"""
-module Internal
-using ..ModuleTools
-@dse
-# TODO: Note to self: we might be a bit more systematic here -- these are
-# "pre-includes" (might go into bits/), contrasting to "post-includes" (which
-# may stay in misc/)
-@inc_dir types accessors misc
-@export_locals
-end
+    # Exports
+    $(EXPORTS)
+    """
+    module Internal
+        using ..ModuleTools
+        @dse
+        # TODO: Note to self: we might be a bit more systematic here -- these are
+        # "pre-includes" (might go into bits/), contrasting to "post-includes" (which
+        # may stay in misc/)
+        @inc_dir types accessors misc
+        @export_locals
+    end
 
-using .Internal
+    using .Internal
 
-@inc_dir types accessors
-@export_locals
+    @inc_dir types accessors
+    @export_locals
 end
 
 # the modules depend on each other so we have to inject the stuff like this
```

Comment:
I added these spaces to make it more readable.. I didn't know where the module begins or ends

Reply:
Is there perhaps a JuliaFormatter setting for that? If this is not reproducible we'll get the PRs pwned by formatchecker forever.

Reply:
will use the new formatter settings